Documentation/upgrading.md

```diff
@@ -1,20 +1,22 @@
 # Upgrading self-hosted Kubernetes
 
-"Self-hosted" Kubernetes clusters run the apiserver, scheduler, controller-manager, kube-dns, kube-proxy, and flannel or calico as pods, like ordinary applications (except with taint tolerations). This allows upgrades to be performed in-place using (mostly) `kubectl` as an alternative to re-provisioning.
+"Self-hosted" Kubernetes clusters run the apiserver, scheduler, controller-manager, kube-dns, kube-proxy, and flannel or calico as pods, like ordinary applications. This allows upgrades to be performed in-place using (mostly) `kubectl`, as an alternative to re-provisioning.
 
 Let's upgrade a Kubernetes v1.6.6 cluster to v1.6.7 as an example.
 
 ## Status
 
-bootkube does not provide in-place patch diffs or migrations between releases. In-place upgrades are inherently more dangerous than a blue/green cluster deployments. Evaluate whether an in-place upgrade is appropriate for your Kubernetes cluster and risk tolerance before beginning. Failover important workloads to another cluster in advance.
+The process of in-place upgrading a self-hosted Kubernetes cluster can be straight-forward, but there may be complex underlying changes that affect your cluster. In most cases, patch version upgrades (e.g. v1.6.6 to v1.6.7) are safe.
+
+Evaluate if an in-place cluster is appropriate for your Kubernetes cluster and risk tolerance. You may wish to test the in-place upgrade on a development cluster, failover important workloads to another cluster, or be prepared to handle unforseen issues.
```

Comment:
I like the change. One nit: "Before beginning an upgrade, is is recommended to evaluate if an in-place..."

Reply:
fixed